src/libraries/Microsoft.PowerFx.Connectors/OpenApiParser.cs

```diff
@@ -427,7 +427,20 @@ private static void ValidateSupportedOpenApiParameters(OpenApiOperation op, ref
         internal static (List<ConnectorFunction> connectorFunctions, List<ConnectorTexlFunction> texlFunctions) ParseInternal(ConnectorSettings connectorSettings, OpenApiDocument openApiDocument, ConnectorLogger configurationLogger = null)
         {
             List<ConnectorFunction> cFunctions = GetFunctionsInternal(connectorSettings, openApiDocument, configurationLogger).Where(f => !f.IsInternal || connectorSettings.IncludeInternalFunctions).ToList();
-            List<ConnectorTexlFunction> tFunctions = cFunctions.Select(f => new ConnectorTexlFunction(f)).ToList();
+            List<ConnectorTexlFunction> tFunctions = cFunctions.Select(f =>
+            {
+                try
+                {
+                    return new ConnectorTexlFunction(f);
+                }
+                catch (Exception)
+                {
+                    // Some exception is possible when evaluating the return type as it internally calls ToConnectorType()
+                    // which can throw on some enum types of unknown schema types like 'file' (known limitation)
+                    // If that case, we allow the creation of ConnetorTexlFunction object w/o a return type (will be DType.UntypedObject)
```

Comment:
Swallowing the exception is always dangerous and leads to supportability issues.   Caller gets back emtpy texlfunction and won't know why. 

Can we do something to capture these, like attach the error to the return object?

Reply:
Will set function as not supported and notSupportedReason = exception.Message